webservice/components/webservice_component.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import requests
+
+from odoo.addons.component.core import AbstractComponent, Component
+
+
+class BaseWebServiceAdapter(AbstractComponent):
```

Comment:
can you split this 2 classes in 2 files? (`base_adapter.py` and `request_adapter.py` shall be good)